pkg/occlient/occlient.go

```diff
@@ -2158,7 +2190,7 @@ func getContainerPortsFromStrings(ports []string) ([]corev1.ContainerPort, error
 }
 
 // CreateBuildConfig creates a buildConfig using the builderImage as well as gitURL.
-func (c *Client) CreateBuildConfig(commonObjectMeta metav1.ObjectMeta, builderImage string, gitURL string) (buildv1.BuildConfig, error) {
+func (c *Client) CreateBuildConfig(commonObjectMeta metav1.ObjectMeta, builderImage string, gitURL string, inputEnvVars []corev1.EnvVar) (buildv1.BuildConfig, error) {
```

Comment:
envVars not inputEnvVars, keep the variable names consistent across functions so dev's aren't confused